### DISCUSSION TITLE
bidirectional lane type not usable

### DISCUSSION THREAD

**Discussion by omia** [carla-simulator/carla#5119] [Q&A]
I created a map in roadrunner and in the Carla import all bidirectional lanes are not usable.
The waypoint are not connected.
![carla](https://user-images.githubusercontent.com/7889102/151707003-5e50f843-fbb7-48d4-99a0-51103bb1be1f.png)
It should work like the roadrunner map.
![roadrunner](https://user-images.githubusercontent.com/7889102/151707052-3fa5c024-82df-4fff-b5e8-6463e5835982.png)

**Comment by manuelmaior29** [comment]
I have also experienced this problem and I didn't manage to find a proper solution, but from what I tested, it seems that this problem appears on road networks that are not closed loops. As a workaround you might try to make your road network a closed loop and see if the issue is fixed.
